google-services-plugin/README.md

```diff
@@ -21,7 +21,7 @@ Apply the plugin in your app's build.gradle:
 
 ```
 plugins {
-    id 'com.google.gms.google-services' version '4.2.0'
+    id 'com.google.gms.google-services' version '4.3.1'
```

Comment:
Won't this be 4.3.2 soon with @davidmotson's change?

Reply:
Fixed!